test/integration/controller/core/capacitycontroller_test.go

```diff
@@ -63,12 +58,7 @@ var _ = ginkgo.Describe("Capacity controller", func() {
 	})
 
 	ginkgo.AfterEach(func() {
-		if ns != nil {
-			err := k8sClient.Delete(ctx, ns)
-			if !errors.IsNotFound(err) {
-				gomega.Expect(err).ToNot(gomega.HaveOccurred())
-			}
-		}
+		gomega.Expect(framework.DeleteNamespace(ctx, k8sClient, ns)).ToNot(gomega.HaveOccurred())
```

Comment:
Just gomega lingo

```suggestion
		gomega.Expect(framework.DeleteNamespace(ctx, k8sClient, ns)).To(gomega.Succeed())
```

The alternate implementation is:

```golang
err := framework.DeleteNamespace(ctx, k8sClient, ns)
gomega.Expect(err).ToNot(gomega.HaveOcurred())
```

Reply:
done.